src/script/api/Doxyfile_GS.in

```diff
@@ -211,15 +207,13 @@ SKIP_FUNCTION_MACROS   = YES
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 TAGFILES               =
-GENERATE_TAGFILE       = ../../../objs/openttd_nogo.tag
+GENERATE_TAGFILE       = ${CPACK_BINARY_DIR}/docs/openttd_nogo.tag
```

Comment:
lol, and here it is "nogo" ... I must have been drunk when making this :D Please also change ;)